ADBench/plot_graphs.py

```diff
@@ -99,7 +130,7 @@ def read_vals(objective, graph_files, tool):
     t_objective_vals = list(map(lambda t: t[1][0], info_sorted))
     t_jacobian_vals = list(map(lambda t: t[1][1], info_sorted))
 
-    return (n_vals, t_objective_vals, t_jacobian_vals)
+    return (n_vals, t_objective_vals, t_jacobian_vals, violation_vals)
```

Comment:
We need to sort violation_vals as I proposed in the commit https://github.com/awf/ADBench/commit/cecc228fa21929955b58a921c1d7799627e9b87e

Reply:
Well spotted!  Thank you.

Reply:
Done